README.md

```diff
@@ -88,159 +93,12 @@ stripe.balance.retrieve({
-  * [`update(refundId[, params])`](https://stripe.com/docs/api/node#update_refund)
-  * [`retrieve(refundId)`](https://stripe.com/docs/api/node#retrieve_refund)
- * skus
-  * [`create(params)`](https://stripe.com/docs/api/node#create_sku)
-  * [`list([params])`](https://stripe.com/docs/api/node#list_skus)
-  * [`update(skuId[, params])`](https://stripe.com/docs/api/node#update_sku)
-  * [`retrieve(skuId)`](https://stripe.com/docs/api/node#retrieve_sku)
-  * [`del(skuId)`](https://stripe.com/docs/api/node#delete_sku)
- * subscriptions
-  * [`create(params)`](https://stripe.com/docs/api/node#create_subscription)
-  * [`list([params])`](https://stripe.com/docs/api/node#list_subscription)
-  * [`update(subscriptionID[, params])`](https://stripe.com/docs/api/node#update_subscription)
-  * [`retrieve(subscriptionId)`](https://stripe.com/docs/api/node#retrieve_subscription)
-  * [`del(subscriptionId)`](https://stripe.com/docs/api/node#delete_subscription)
- * tokens
-  * [`create(params)`](https://stripe.com/docs/api/node#create_card_token)
-  * [`retrieve(tokenId)`](https://stripe.com/docs/api/node#retrieve_token)
- * threeDSecure
-  * [`create(params)`](https://stripe.com/docs/api/node#create_three_d_secure)
- * transfers
-  * [`create(params)`](https://stripe.com/docs/api/node#create_transfer)
-  * [`list([params])`](https://stripe.com/docs/api/node#list_transfers)
-  * [`retrieve(transferId)`](https://stripe.com/docs/api/node#retrieve_transfer)
-  * [`update(transferId[, params])`](https://stripe.com/docs/api/node#update_transfer)
-  * [`reverse(transferId[, params])`](https://stripe.com/docs/api/node#create_transfer_reversal)
-  * `cancel(transferId)` (Deprecated -- use `reverse`)
-  * [`listTransactions(transferId[, params])`](https://stripe.com/docs/api/curl#list_transfers)
-  * `setMetadata(transferId, metadataObject)` ([metadata info](https://stripe.com/docs/api/node#metadata))
-  * `setMetadata(transferId, key, value)`
-  * `getMetadata(transferId)`
- * bitcoinReceivers
-  * [`create(params)`](https://stripe.com/docs/api/node#create_bitcoin_receiver)
-  * [`retrieve(receiverId)`](https://stripe.com/docs/api/node#retrieve_bitcoin_receiver)
-  * [`list([params])`](https://stripe.com/docs/api/node#list_bitcoin_receivers)
-  * `getMetadata(receiverId)`
-
-## Configuration
+## Documentation
 
- * `stripe.setApiKey(' your secret api key ');`
- * `stripe.setTimeout(20000); // in ms` (default is node's default: `120000ms`)
+See the [Node API docs](https://stripe.com/docs/api/node#intro).
 
-## More information / wikis
+## More Information
 
- * **[In-depth Documentation](https://stripe.com/docs/api/node)**
- * [Version 2 Overview](https://github.com/stripe/stripe-node/wiki/Version-2)
  * [REST API Version](https://github.com/stripe/stripe-node/wiki/REST-API-Version)
  * [Error Handling](https://github.com/stripe/stripe-node/wiki/Error-Handling)
  * [Passing Options](https://github.com/stripe/stripe-node/wiki/Passing-Options)
```

Comment:
should we remove those too? The error handling should at least point to https://stripe.com/docs/api/node#errors instead